docs/compose-file.md

```diff
@@ -120,6 +120,15 @@ Override the default command.
 
     command: bundle exec thin -p 3000
 
+The command can also be a list, in a manner similar to [dockerfile](https://docs.docker.com/engine/reference/builder/#cmd):
+
+    command:
+        - bundle
+        - exec
+        - thin
+        - -p
+        - 3000
```

Comment:
Maybe a minor point, but I would personally use the `[]` form of a list for something like this:

```yaml
command: [bundle, exec, thin, -p, 3000]
```